musicbot/bot.py

```diff
@@ -2637,16 +2637,18 @@ async def handle_vc_inactivity(self, guild: discord.Guild) -> None:
             )
         except asyncio.TimeoutError:
             # could timeout after a disconnect.
-            if guild.me.voice and guild.me.voice.channel:
+            if guild.voice_client and isinstance(
+                guild.voice_client.channel, (discord.VoiceChannel, discord.StageChannel)
```

Comment:
Did you mean to put whitespace here? Or is mobile just being funny?

Reply:
Yes. think it was black formatting.